ucsb-cs156-f23/team03-f23-5pm-4
# Create Page for MenuItemReview plus tests and stories

Create Page for MenuItemReview, with tests and stories
<img width="1671" alt="Screenshot 2023-11-08 at 5 37 55 PM" src="https://github.com/ucsb-cs156-f23/team03-f23-5pm-4/assets/46696490/4ff83390-53e5-4d3a-b453-19f91bcce79a">

[Storybook](https://ucsb-cs156-f23.github.io/team03-f23-5pm-4/prs/94/storybook/?path=/story/components-menuitemreview-menuitemreviewform--create)

Testing instructions:

1. log into [dokku deployment](https://team03-mitali-g-dev.dokku-04.cs.ucsb.edu)
2. navigate to [create page](https://team03-mitali-g-dev.dokku-04.cs.ucsb.edu/menuitemreview/create)
3. input fields and press create
4. navigate to [swagger](https://team03-mitali-g-dev.dokku-04.cs.ucsb.edu/swagger-ui/index.html)
5. verify that GET all endpoint includes the item you just created

Closes #40
